### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by andrerochow** [zengqunzhao/EfficientFace#31]
Hi, 

did you use the same normalization for training the pre-trained and fine-tuned models on all different datasets?

"normalize = transforms.Normalize(mean=[0.57535914, 0.44928582, 0.40079932], std=[0.20735591, 0.18981615, 0.18132027])"

**Comment by andrerochow** [comment]
I just found the normalization values for the other datasets in the [test.py](https://github.com/zengqunzhao/EfficientFace/blob/master/test%26vis/test.py#L33-L41) file:

    # RAF-DB
    normalize = transforms.Normalize(mean=[0.57535914, 0.44928582, 0.40079932],
                                      std=[0.20735591, 0.18981615, 0.18132027])
    # # AffectNet
    # normalize = transforms.Normalize(mean=[0.55391484, 0.43522123, 0.3821877],
    #                                   std=[0.24837189, 0.21625527, 0.20221159])
    # # CAER-S
    # normalize = transforms.Normalize(mean=[0.35964426, 0.22359873, 0.19040863],
    #                                   std=[0.15188067, 0.11678473, 0.1103445])
